CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -2037,6 +2037,8 @@ protected function processFormSubmission($contactID) {
     // fix currency ID
     $this->_params['currencyID'] = CRM_Core_Config::singleton()->defaultCurrency;
 
+    CRM_Contribute_Form_AbstractEditPayment::formatCreditCardDetails($this->_params);
```

Comment:
@eileenmcnaughton we cannot call beginPostProcess() here as it extend different parent class also there are lots of formvalue attributes which are formatted here (and to move into common formatting function will need lots of refactoring changes) so I thought its better format the credit card details only with the help of this function.